### PR TITLE
[14.0][MIG] stock_move_location: Migration to 14.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa
TT26454

ping @sergio-teruel @pedrobaeza

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

File: stock_move_location/data/stock_quant_view.xml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-        id="wiz_stock_quant_location_action"
-        name="Move to location..."
-        res_model="wiz.stock.move.location"
-        binding_model="stock.quant"
-        view_mode="form"
-        context="{'origin_location_disable': True}"
-        target="new"
-        groups="stock.group_stock_user"
-    />
+    <data>
```
Comment (pedrobaeza) [inline stock_move_location/data/stock_quant_view.xml]: Don't add this key for not re-indenting everything

File: stock_move_location/wizard/stock_move_location.xml [hunk 2]
```diff
@@ -1,153 +1,170 @@
...
-                            widget="one2many_list"
-                            mode="tree,kanban"
-                        >
-                            <tree
-                                string="Inventory Details"
-                                editable="bottom"
-                                decoration-info="move_quantity != max_quantity"
-                                decoration-danger="(move_quantity &lt; 0) or (move_quantity > max_quantity)"
-                                create="0"
+    <data>
```
Comment (pedrobaeza) [inline stock_move_location/wizard/stock_move_location.xml]: Don't add this key for not re-indenting everything. It's not needed.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by joao-p-marques** [comment]
Changes made @pedrobaeza

File: stock_move_location/README.rst [hunk 3]
```diff
@@ -92,12 +92,15 @@ Contributors
...
 * Mathieu Vatel <mathieu@julius.fr>
 * Mykhailo Panarin <m.panarin@mobilunity.com>
-* Sergio Teruel <sergio.teruel@tecnativa.com>
 * Joan Sisquella <joan.sisquella@forgeflow.com>
 * Jordi Ballester Alomar <jordi.ballester@forgeflow.com>
 * Lois Rilo <lois.rilo@forgeflow.com>
 * Héctor Villarreal <hector.villarreal@forgeflow.com>
 
+* Tecnativa <tecnativa.com>
+  * Sergio Teruel
```
Comment (pedrobaeza) [inline stock_move_location/README.rst:101]: This hasn't been added correctly for the syntax: the empty line must be between these lines, not above

Comment (joao-p-marques) [reply stock_move_location/README.rst:101]: Oh sorry, MB. Already fixed.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**